src/pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.Ref.builds

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <HasRuntimePackages>false</HasRuntimePackages>
+  </PropertyGroup>
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+
+  <ItemGroup>
+    <Project Include="$(MSBuildProjectName).pkgproj" />
```

Comment:
Consider putting this in a folder of the same name rather than putting more than one project in a single directory.

Reply:
Yeah, having it in the same dir does seem confusing, so I'll move it.

In the future I want to get rid of it and add tooling support to use one `Microsoft.NETCore.App.builds` for all of these packs (similar to how `runtime.<rid>.*` packages are created), but there's no need to break convention now to do that.

My motivation is thinking about how it'd look "unrolled":
* `external`
  * `Microsoft.NETCore.App`
    * `Microsoft.NETCore.App.depproj`
      * Moved from package-specific `src` dir to follow a corefx-like convention. Makes the reuse easier to see.
  * `Microsoft.WindowsDesktop.App`
    * `Microsoft.WindowsDesktop.App.depproj`
* Each of these has `.builds` and `.pkgproj` (and any package-specific stuff like `.targets`):
* `Microsoft.NETCore.App`
* `Microsoft.NETCore.App.Ref`
* `Microsoft.NETCore.App.Runtime`
* `Microsoft.NETCore.App.Host`
* `Microsoft.WindowsDesktop.App.Ref`
* `Microsoft.WindowsDesktop.App.Runtime`
* `Microsoft.WindowsDesktop.App.Host`
* \+ netstandard targeting, runtime, apphost packs.

Reply:
Feel free to kill the whole `*.builds` convention if you come up with something simpler.